entmax/__init__.py

```diff
@@ -1,19 +1,25 @@
-__version__ = "1.3.dev0"
+__version__ = "1.4.dev0"
```

Comment:
Actually, keep it as 1.3.dev0. The last release was 1.2. We only need to change the `__version__` immediately after we release a new version on pip.